include/fmt/core.h

```diff
@@ -2675,7 +2675,7 @@ void vformat_to(buffer<Char>& buf, basic_string_view<Char> fmt,
                 typename vformat_args<Char>::type args, locale_ref loc = {});
 
 FMT_API void vprint_mojibake(std::FILE*, string_view, format_args);
-#ifndef _WIN32
+#if !defined(_WIN32) || defined(FMT_WINDOWS_NO_WCHAR)
```

Comment:
Can we not change this? It's weird to disable overload that doesn't even use `wchar_t`.

Reply:
If we don't have wchar, we don't have `vprint_mojibake` either, and the compiler complains about the lack of a definition.

Reply:
Actually maybe I'm wrong, let me investigate

Reply:
This only happens when `is_utf8()` returns `false` but this is the error:

```
fmt/core.h:2894: undefined reference to `fmt::v10::detail::vprint_mojibake(_iobuf*, fmt::v10::basic_string_view<char>, fmt::v10::basic_format_args<fmt::v10::basic_format_context<fmt::v10::appender, char> >)'
```

Reply:
This is the wrong fix though because this will just discard the output. The correct fix is to enable the `vprint_mojibake` definition in `format-inl.h`.

Reply:
Thanks, done